README.md

```diff
@@ -183,10 +183,33 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
 - `repo`\
   The GitHub repository name.
 
+###  Authentication keys
+
+Keys must be provided for one of the following authentication methods:
```

Comment:
If I am a new user and I arrive here, I don't know which one to pick.
In this section, just after the list with the 2 options, we could say something to help the user choose.

Maybe something along these lines:

```
Which authentication method to choose?

The advantage of using a Personal Access Token is that it is simpler to setup,
while the advantage of a GitHub App is that it has a more generous rate limit.
See section [#Caveat: GitHub rate limiting] for details.
```

Reply:
Fixed as suggested. Thanks